src/django/api/views.py

```diff
@@ -2287,6 +2288,7 @@ def get_parent_company_options(self, request):
 @api_view(['GET'])
 @permission_classes([AllowAny])
 @renderer_classes([MvtRenderer])
+@cache_page(settings.TILE_CACHE_MAX_AGE_IN_SECONDS)
```

Comment:
Searching for `cache_page` in the Django docs you end up on https://docs.djangoproject.com/en/2.2/topics/cache/, which discusses the caching framework of which `cache_page` is a part. Based on that documentation an the comments in https://github.com/django/django/blob/ce97960e34baf7db865b5424d3c61274a9f88919/django/views/decorators/cache.py#L8 I think this decorator caches the object using Django machinery, which is not what we want.

The docs say:
>Local-memory caching
>
>This is the default cache if another is not specified in your settings file.

This leads me to believe that when we use the `cache_page` decorator we are be putting tiles into a local memory cache.

It looks like https://docs.djangoproject.com/en/2.2/topics/http/decorators/#django.views.decorators.cache.cache_control may be more appropriate for what we are trying to do. 


````

Reply:
Ah! That's a great catch. Yes, `cache_control` is more what we want. Will switch to it.

Reply:
Nice catch.